mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -1,65 +1,101 @@
 pipeline {
```

Comment:
@whchung How would you like this job to be? It looks like this one will replace the original `MLLIR-MIopen-Resnet50-nightly` job. Would you prefer this to be continue to be a nightly job?